1-JavaScript/Week3/homework/ex2-dogYears.js

```diff
@@ -14,8 +14,10 @@ calculate it!
    ages.
 -----------------------------------------------------------------------------*/
 
-function calculateDogAge(/* parameters go here */) {
+function calculateDogAge(age) {
   // TODO complete this function
+  age = age * 7;
```

Comment:
I think you can remove the `TODO` comments once you have implemented the logic. Also, in this case, I would recommend to create a new variable here(if needed) instead of mutating the function argument. It would be much cleaner. For example,

```
function calculateDogAge(age) {
  const dogAge = age * 7;
  return `Your doggie is ${dogAge} years old in dog years!`;
}
```

or

```
function calculateDogAge(age) {
  return `Your doggie is ${age * 7} years old in dog years!`;
}
```